src/main/utils/svg/multiline.tsx

```diff
@@ -31,7 +31,7 @@ export class Multiline extends Component<Props, State> {
   spaceWidth = 0;
   wordsWithComputedWidth = [] as { word: string; width: number }[];
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
```

Comment:
What is this change for? Also, both of these functions are apparently deprecated.